Makefile

```diff
@@ -1,22 +1,22 @@
 .PHONY: build release install uninstall clean test reindent reformat
 
 build:
-	jbuilder build @install --dev
+	dune build @install
 
 release:
-	jbuilder build @install
+	dune build @install --profile=release
 
 install:
-	jbuilder install
+	dune install
 
 uninstall:
-	jbuilder uninstall
+	dune uninstall
 
 clean:
-	jbuilder clean
+	dune clean
 
 test:
-	jbuilder runtest
+	dune runtest --profile=release
```

Comment:
Is this to avoid failing on warnings?

Reply:
No, I checked other packages and used it this way, I can remove it if it has drawbacks

Reply:
Then let's do without, so we have the tests failing if there are new warnings